staging/src/k8s.io/apiextensions-apiserver/test/integration/registration_test.go

```diff
@@ -105,12 +82,7 @@ func NewNamespacedCustomResourceVersionedClient(ns string, client dynamic.Interf
 }
 
 func NewNamespacedCustomResourceClient(ns string, client dynamic.Interface, crd *apiextensionsv1beta1.CustomResourceDefinition) dynamic.ResourceInterface {
```

Comment:
Move this to `resources.go` instead of here?

Reply:
Same for `NewNamespacedCustomResourceVersionedClient`

Reply:
don't want to promote these helpers. Instead I moved them into helpers.go and made them private. Similarly, some testserver helpers only used in k8s.io/apiextensions-apiserver I also moved into helpers.go.

Reply:
:+1: